app/src/test/java/org/schabi/newpipe/settings/ContentSettingsManagerTest.kt

```diff
@@ -166,6 +166,18 @@ class ContentSettingsManagerTest {
         assertFalse(contains)
     }
 
+    @Test
+    fun `IsValid must return false if the zip has not the two necessary files`() {
```

Comment:
There are 2 things tested here so it should be seperated into 2 tests

Reply:
Okay